packages/gatsby-theme-docs/src/components/limit.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ContentNotifications } from '@commercetools-docs/ui-kit';
+import { useLimitByName } from '../hooks/use-limits';
+
+const Limit = (props) => {
+  const limit = useLimitByName(props.name);
+
+  if (!limit) {
+    const message = `${props.name} limit is not found.`;
+
+    if (process.env.NODE_ENV !== 'production') {
+      return <ContentNotifications.Error>{message}</ContentNotifications.Error>;
+    }
+
+    throw new Error(message);
+  }
+
+  return (
+    <>
```

Comment:
Should we use a `span` here?

Reply:
I don't think it's necessary.

Reply:
Not now since we don't have any special styling instruction.  

But I can imagine that we later add more metadata to a limit (e.g. a label or a topic), generate a central overview table and have the component link to it.   But to start off and get the migration moving just text is fine.